### ISSUE TITLE
Incorrect ordering of navigation/header for screen reader

### ISSUE THREAD

**Issue by ravicodelabs** [datascijedi/website#136]
The navigation/header items should be traversed form left-to-right by a screen reader, but currently it jumps to the search bar earlier in the process. 

See issue filed with Quarto for details:

https://github.com/quarto-dev/quarto-cli/issues/7058

(Moved from https://github.com/datascijedi/website/issues/115.)

**Comment by ravicodelabs** [comment]
Issue still present as of Quarto v1.5.57.

**Comment by ravicodelabs** [comment]
Issue still present as of pre-release v1.6.15.
